plugins/auth-backend/api-report.md

```diff
@@ -59,9 +59,17 @@ export type Auth0ProviderOptions = {
   };
 };
 
+// @public
+export type AuthContext = {
+  tokenIssuer: TokenIssuer;
+  catalogIdentityClient: CatalogIdentityClient;
+  logger: Logger_2;
+};
+
 // @public
 export type AuthHandler<TAuthResult> = (
   input: TAuthResult,
+  context?: AuthContext,
```

Comment:
Tbh I think we might as well make this required straight away. It's a bit awkward to consume otherwise.

It would make it a minor change, but It's enough with some quick guidance that the context needs to be passed here in addition to the `signIn.resolver`

Reply:
Would be fine for me - our intention was just to not make it a breaking change 😉 Up to you, we could make it mandatory now within this PR or in a separate one.

Reply:
Maybe I'm being silly but, if we wanted to do it gradually, we could probably do

```ts
/**
 * Blah
 *
 * @public
 * @deprecated
 */
export type LegacyAuthHandler<TAuthResult> = (input: TAuthResult) => Promise<AuthHandlerResult>

export type AuthHandler<TAuthResult> = LegacyAuthHandler | ((input: TAuthResult, context: AuthContext) => Promise<AuthHandlerResult>);
```

Reply:
After some discussion: Let's just break this immediately and make the `context` argument required. Remember to update the changeset.

Reply:
Okay, will update files 👍

Reply:
Updated all providers! 👍